src/reports/resources/demo.js

```diff
@@ -2,10 +2,10 @@ CONSTANTS = {"PI": "real", "E": "real", "TRUE": "bool", "FALSE": "bool"}
 
 FUNCTIONS = {}
 
-"+ - * / pow copysign fdim fmin fmax fmod hypot remainder".split(" ").forEach(function(op) {
+"+ - * / pow copysign fdim fmin fmax fmod hypot remainder atan2".split(" ").forEach(function(op) {
     FUNCTIONS[op] = [["real", "real"], "real"];
 });
-("fabs sqrt exp log sin cos tan asin acos atan sinh cosh tanh asinh acosh atanh " +
+("fabs sqrt exp log sin cos tan asin acos atan sinh cosh tanh asinh acosh atanh" +
```

Comment:
Please don't change this. It's a little non-obvious, but by removing this space you actually defined a function called `atanhcbrt` instead of two functions called `atanh` and `cbrt`. More generally, please review the "Files changed" tab (or the equivalent inside VS Code) and make sure you don't have any changes that you didn't intend.